hnn_core/tests/test_dipole.py

```diff
@@ -65,36 +81,40 @@ def test_dipole_simulation():
     with pytest.raises(TypeError, match="record_isoma must be bool, got int"):
         simulate_dipole(net, n_trials=1, record_vsoma=False, record_isoma=0)
 
-    trial, n_trials, gid = 0, 2, 7
-    n_times = np.arange(0., params['tstop'] + params['dt'], params['dt']).size
-    mpi_net = Network(params)
-    with MPIBackend(n_procs=None, mpi_cmd='mpiexec'):
-        simulate_dipole(mpi_net, n_trials=n_trials, record_vsoma=True,
-                        record_isoma=True)
-        assert len(mpi_net.cell_response.vsoma) == n_trials
-        assert len(mpi_net.cell_response.isoma) == n_trials
-        assert len(mpi_net.cell_response.vsoma[trial][gid]) == n_times
-        assert len(mpi_net.cell_response.isoma[
-                   trial][gid]['soma_gabaa']) == n_times
-
-    joblib_net = Network(params)
-    with JoblibBackend(n_jobs=1):
-        simulate_dipole(joblib_net, n_trials=n_trials, record_vsoma=True,
-                        record_isoma=True)
-        assert len(joblib_net.cell_response.vsoma) == n_trials
-        assert len(joblib_net.cell_response.isoma) == n_trials
-        assert len(joblib_net.cell_response.vsoma[trial][gid]) == n_times
-        assert len(joblib_net.cell_response.isoma[
-                   trial][gid]['soma_gabaa']) == n_times
+
+@requires_mpi4py
+def test_cell_response_backends(run_hnn_core):
+    """Test cell_response outputs across backends."""
+
+    # reduced simulation has n_trials=2
+    trial_idx, n_trials, gid = 0, 2, 7
+    _, joblib_net = run_hnn_core(backend='joblib', n_jobs=1, reduced=True,
+                                 record_isoma=True, record_vsoma=True)
+    _, mpi_net = run_hnn_core(backend='mpi', n_jobs=1, reduced=True,
```

Comment:
The "n_jobs" parameter doesn't do anything for MPIBackend. It defaults to running JoblibBackend. What about setting "n_procs=2"?

For a future PR that will help avoid this, I would suggest using "parallel" to replace "n_jobs" and "n_procs".